src/sign_workflow/signer_pgp.py

```diff
@@ -19,7 +19,7 @@
 """
 
 
-class Signer:
+class SignerPGP:
```

Comment:
Create a common parent class for both signers and move any common logic (e.g. git repo) into it.